Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -102,12 +102,7 @@ public void setTargetSpeedForRange(double rangeInInches) {
     }
     
     public boolean isWheelAtSpeed() {
-        // Speed is in native units and can be negative
-        double currentSpeed = Math.abs(super.getSpeed());
-        if (currentSpeed <= 1) {
-            return false;
-        }
-        return currentSpeed >= super.getTargetSpeed() * wheelSpeedThresholdPercentage.get();
+        return Math.abs(getSpeed() - getTargetSpeed()) <= getTargetSpeed() * wheelSpeedThresholdPercentage.get();
```

Comment:
Looks reasonable, just to be safe let's make sure we test it out on the robot with a dry-run of autonomous in the pit / practice area before running this change in competition.